scripts/plot_noise.py

```diff
@@ -78,30 +111,49 @@ def plot(savefig=False, **kwargs):
     plt.subplots_adjust(top=0.85)
     if savefig:
         plt.savefig("galactic_noise.png", bbox_inches="tight")
-    plt.show()
+    #plt.show()
 
-if __name__ == "__main__":
-    import argparse
-
-    parser = argparse.ArgumentParser(
-        description="Parser to select which noise quantity to plot. \
-        To Run: ./plot_noise.py \
-        Add --lst <int> for galactic noise. i.e ./plot_noise.py --lst 18 --savefig."
-        )
-    parser.add_argument(
-        "--lst",
-        default=18.0,
-        type=float,
-        help="lst for Local Sideral Time, galactic noise is variable with LST and maximal for 18h.",
-        )
-    parser.add_argument(
-        "--savefig",
-        action="store_true",
-        default=False,
-        help="don't add galactic noise.",
-    )
+def main():
+    parser = argparse.ArgumentParser(description="Plot function with command line arguments")
+    parser.add_argument('--savefig', action='store_true', help="Flag to save the figure")
+    parser.add_argument('--du_type', type=str, default='GP300_nec', help="Type of du")
```

Comment:
I think we should agree on a uniform default for the du_type accross all functions. Here GP300_nec is being used instead of GP300 as in efield2voltage

Reply:
Yes the default is GP300 (fixed)